Move Passthrough Config from Automate to JSON Reporter (#5430)
chef/inspec: docs-chef-io/content/inspec/reporters.md

### json

> This reporter includes all information about the profiles and test results in standard json format. You may optionally pass through arbitrary structured JSON data via a json config with the `--config` parameter like the example below.
> 
> Example config:
> 

Reviewer: ```suggestion
This reporter includes all information about the profiles and test results in standard JSON format. You may optionally pass through arbitrary structured JSON data by setting a JSON configuration with the `--config` parameter. 

For example:

```

Author: That's fine with me @IanMadd. Committed and back to you for final review!

Author: Also updated sign-off for DCO. I guess don't use the Github UI if you care about that for Chef review process! 😆
